lib/spack/spack/build_systems/cmake.py

```diff
@@ -163,15 +170,15 @@ def _std_args(pkg):
         define = CMakePackage.define
         args = [
             '-G', generator,
-            define('CMAKE_INSTALL_PREFIX', pkg.prefix),
+            define('CMAKE_INSTALL_PREFIX', pkg.prefix.replace('\\', '/')),
             define('CMAKE_BUILD_TYPE', build_type),
         ]
 
         # CMAKE_INTERPROCEDURAL_OPTIMIZATION only exists for CMake >= 3.9
         if pkg.spec.satisfies('^cmake@3.9:'):
             args.append(define('CMAKE_INTERPROCEDURAL_OPTIMIZATION', ipo))
 
-        if primary_generator == 'Unix Makefiles':
+        if generator == 'Unix Makefiles':
```

Comment:
I think we'll want to keep `if primary_generator == 'Unix Makefiles':` - if I understand correctly this won't affect operation on Windows, and `primary_generator` is `generator` stripped of all suffixes (so will match something like `Unix Makefiles: A secondary generator`

Reply:
Addressed.